awx/ui_next/src/screens/Template/Survey/SurveyQuestionAdd.jsx

```diff
@@ -9,23 +9,48 @@ export default function SurveyQuestionAdd({ survey, updateSurvey }) {
   const match = useRouteMatch();
 
   const handleSubmit = async question => {
+    const formData = { ...question };
     try {
-      if (survey.spec?.some(q => q.variable === question.variable)) {
+      if (survey.spec?.some(q => q.variable === formData.variable)) {
         setFormError(
           new Error(
-            `Survey already contains a question with variable named “${question.variable}”`
+            `Survey already contains a question with variable named “${formData.variable}”`
           )
         );
         return;
       }
-      if (question.type === 'multiselect') {
-        question.default = question.default
+      let choices = '';
+      let defaultAnswers = '';
+      if (
+        formData.type === 'multiselect' ||
+        formData.type === 'multiplechoice'
+      ) {
+        formData.formattedChoices.forEach(({ choice, isDefault }, i) => {
+          choices =
+            i === formData.formattedChoices.length - 1
+              ? choices.concat(`${choice}`)
+              : choices.concat(`${choice}\n`);
+          if (isDefault) {
+            defaultAnswers =
+              i === formData.formattedChoices.length - 1
+                ? defaultAnswers.concat(`${choice}`)
+                : defaultAnswers.concat(`${choice}\n`);
+          }
+        });
+        formData.default = defaultAnswers;
+        formData.choices = choices;
+      }
+      delete formData.formattedChoices;
+
+      if (formData.type === 'multiselect') {
+        formData.default = formData.default
           .split('\n')
           .filter(v => v !== '' || '\n')
           .map(v => v.trim())
           .join('\n');
       }
-      const newSpec = survey.spec ? survey.spec.concat(question) : [question];
+      const newSpec = survey.spec ? survey.spec.concat(formData) : [formData];
+
```

Comment:
When this feature was originally put in the new ui there was some talk about whether to add empty strings.  We decided to allow to add them, however, now that we have prompt on launch and workflow prompting work in the new ui I believe that adding empty strings in the default values, or in the choices themselves is bad design because it is very confusing when the user goes to answer the survey questions (imagine a drop down with 1 of the choices being an empty string. That would render as just a blank choice).